js/compiler.js

```diff
@@ -65,10 +65,17 @@ define( [
       var blockPiles = this.generateBlocks( output );
 
       console.log( "blockPiles", blockPiles );
+      if( blockPiles[0].blocks.length === 0 ) {
```

Comment:
I think this can cause problems when your first instruction is `position`. I have to test it.